library/core/src/array/mod.rs

```diff
@@ -198,7 +198,8 @@ impl Error for TryFromSliceError {
 }
 
 #[stable(feature = "try_from_slice_error", since = "1.36.0")]
-impl From<Infallible> for TryFromSliceError {
+#[rustc_const_unstable(feature = "const_trait_impl", issue = "67792")]
```

Comment:
```suggestion
#[rustc_const_unstable(feature = "const_try", issue = "74935")]
```

Reply:
Should I put everything in this PR under that feature?

Reply:
Yes